### ISSUE TITLE
rotate genome label

### ISSUE THREAD

**Issue by sbbmu** [mrcuizhe/rNGCircos#4]
Thanks for making the great package. I was wondering is there a way to rotate genome labels 90 degrees? I found there is a variable "genomeLabelOrientation", but it seems ignored. I wasn't able to find any instructions either. Please help! Thank you!

**Comment by mrcuizhe** [comment]
> Thanks for making the great package. I was wondering is there a way to rotate genome labels 90 degrees? I found there is a variable "genomeLabelOrientation", but it seems ignored. I wasn't able to find any instructions either. Please help! Thank you!

Yes, **genomeLabelOrientation** is not working, please try **dx** for rotation of genome label

**Comment by sbbmu** [comment]
It looks like `genomeLabelDx` shifts on x axis, not a rotation. I am pursuing something like this.

![image](https://user-images.githubusercontent.com/4334177/181079038-fdab2823-a37f-4b71-8b6b-24fcd8be200e.png)

**Comment by mrcuizhe** [comment]
> It looks like `genomeLabelDx` shifts on x axis, not a rotation. I am pursuing something like this.
> 
> ![image](https://user-images.githubusercontent.com/4334177/181079038-fdab2823-a37f-4b71-8b6b-24fcd8be200e.png)

I'm sorry, this kind of rotation is not supported by NG-Circos, which means I can't add this feature in ineracCircos immediately.
